addons/event/models/event_event.py

```diff
@@ -350,6 +405,11 @@ def _read_group_stage_ids(self, stages, domain, order):
 
     @api.model
     def create(self, vals):
+        # Temporary fix: as ``seats_availability`` and ``date_tz`` are required but stored fields
+        # are computed after create, required constraint cannot be set -> manually computed it
+        # currently
```

Comment:
Maybe you could just refer to method docstring as the comments seem similar.

Reply:
Hagrid.